tests/scripts/all.sh

```diff
@@ -145,6 +159,11 @@ General options:
   -k|--keep-going       Run all tests and report errors at the end.
   -m|--memory           Additional optional memory tests.
      --armcc            Run ARM Compiler builds (on by default).
+     --except           If some components are passed on the command line,
+                        run all the tests except for these components. In
+                        this mode, you can pass shell wildcard patterns tha
+                        component names, e.g. "$0 --except 'test_*'" to
+                        exclude all components that run tests.
```

Comment:
The commit contents and the commit message seem to give a different view of this option. From the commit content it looks like the second `--except` in the commit message is redundant, as usage would be `--except --potentially-other-options-here foo bar baz`.

Reply:
Fixed by rewording the commit message